### PR TITLE
Change the calls that uses the rabbit_json modul

### PR DESCRIPTION
**Pull request by Gsantomaggio**
See :rabbitmq/rabbitmq-common#164 

Change cleanup function

### CODE REVIEW

File: test/federation_mgmt_SUITE.erl [hunk 1]
```diff
@@ -251,14 +251,15 @@ assert_code(CodeExp, CodeAct, Type, Path, Body) ->
...
 
 cleanup(L) when is_list(L) ->
     [cleanup(I) || I <- L];
-cleanup({struct, I}) ->
-    cleanup(I);
-cleanup({K, V}) when is_binary(K) ->
-    {list_to_atom(binary_to_list(K)), cleanup(V)};
+cleanup(M) when is_map(M) ->
+    maps:fold(fun(K, V, Acc) ->
+        Acc#{binary_to_atom(K, latin1) => cleanup(V)}
```
Comment (michaelklishin) [inline test/federation_mgmt_SUITE.erl]: Do we really want to use `latin1` here?

Comment (Gsantomaggio) [inline test/federation_mgmt_SUITE.erl]: Good point:
I got the same from:
https://github.com/rabbitmq/rabbitmq-shovel-management/blob/master/test/http_SUITE.erl#L262
https://github.com/rabbitmq/rabbitmq-tracing/blob/master/test/rabbit_tracing_SUITE.erl#L229

When we refactor these functions we could change it

Comment (michaelklishin) [inline test/federation_mgmt_SUITE.erl]: @Gsantomaggio let's use `utf8`.

Comment (michaelklishin) [inline test/federation_mgmt_SUITE.erl]: …everywhere.

Comment (Gsantomaggio) [inline test/federation_mgmt_SUITE.erl]: done :)!

**Pull request merged**